javascript/newer-JS-features.md

```diff
@@ -222,6 +222,184 @@ const { email, password } = user; //equivalent to saying const email = user.emai
 console.log(email); // renunikhilp@gmail.com
 ```
 
+### Temporal API Features
+
+Temporal API provides better date and time handling capabilities in JavaScript. We'll look at the classes: `PlainDate`, `PlainTime`, and `ZonedDateTime`.
+
+### **PlainDate**
+`PlainDate` represents a calendar date without any associated time or time zone.
+
+```javascript
+const date = Temporal.PlainDate.from('2024-12-25');
+console.log(date.year); // 2023
+console.log(date.month); // 12
+console.log(date.day); // 25
+```
+
+### **PlainTime**
```

Comment:
```suggestion
##### **PlainTime**
```

Reply:
I've added this along with some additional formatting updates in a commit

Reply:
It’s best practice to cite which commit by providing your hash in the replying comment.